code/modules/reagents/reagent_containers/glass/beaker.dm

```diff
@@ -27,10 +28,15 @@
 		var/mutable_appearance/lid = mutable_appearance(icon, lid_icon)
 		add_overlay(lid)
 
-	if(reagents && reagents.total_volume)
-		var/mutable_appearance/filling = mutable_appearance('icons/obj/reagentfillings.dmi', "[icon_state][get_filling_state()]")
-		filling.color = reagents.get_color()
-		add_overlay(filling)
+	if(reagents.total_volume)
```

Comment:
Dream Daemon reported a runtime right here, likely because `reagents` was `null`. Change this to `if(reagents?.total_volume)` so that it first checks if `reagents` exists.

Reply:
done